Errors >>>>
Runtime Error: Error message 2
Result >>>>
Exception1 caught
